### ISSUE TITLE
module 'utils' has no attribute 'cent_norm'

### ISSUE THREAD

**Issue by mett96** [nikitakaraevv/pointnet#15]
When I try to run the notebook 
```
AttributeError                            Traceback (most recent call last)
<ipython-input-16-6f9310cafa62> in <module>
     12 print('######### Dataset class created #########')
     13 print('Number of images: ', len(dset))
---> 14 print('Sample image shape: ', dset[0]['image'].shape)
     15 #print('Sample image points categories', dset[0]['category'], end='\n\n')
     16 

<ipython-input-15-a0ffe3b5dad8> in __getitem__(self, idx)
     36         category = self.files[idx]['category']
     37         with open(img_path, 'r') as f:
---> 38             image1 = read_pts(f)
     39         with open(category, 'r') as f:
     40             category1 = read_seg(f)

<ipython-input-13-6a351433de44> in read_pts(file)
      3 def read_pts(file):
      4     verts = np.genfromtxt(file)
----> 5     return utils.cent_norm(verts)
      6     #return verts
      7 

AttributeError: module 'utils' has no attribute 'cent_norm'
```

**Comment by st5714** [comment]
I had the same problem. module 'utils' has no attribute 'cent_norm'

**Comment by nikitakaraevv** [comment]
Hi @mett96,  @st5714,

Indeed, it doesn't work. 
Thank you for point it out, I will fix the segmentation notebook so it works with the update version of code.
In the meanwhile, you can check out the [classification notebook](https://colab.research.google.com/github/nikitakaraevv/pointnet/blob/master/nbs/PointNetClass.ipynb). There is no such a problem for this one. 

Thanks again,
Nikita

**Comment by normalusername1** [comment]
Hey, do you know when this problem is going to be fixed?

**Comment by normalusername1** [comment]
Did this problem ever end up getting fixed?

**Comment by mserouar** [comment]
Wondering too

**Comment by michaelhasey** [comment]
Hi Nikita, 

Spent a good chunk of time running through your classification model with great results and hoping to do the same with your part segmentation model.  Have you had the chance to fix this issue?  Would be great to implement your model!

Thanks,

Michael

**Comment by bwlstart** [comment]
> Hi @mett96, @st5714,
> 
> Indeed, it doesn't work. Thank you for point it out, I will fix the segmentation notebook so it works with the update version of code. In the meanwhile, you can check out the [classification notebook](https://colab.research.google.com/github/nikitakaraevv/pointnet/blob/master/nbs/PointNetClass.ipynb). There is no such a problem for this one.
> 
> Thanks again, Nikita

Hi @nikitakaraevv ,I also had the same problem when I ran the PointNetSeg. It would be cool if the model could be implemented. Looking forward  to it.

**Comment by Largehare** [comment]
i solve this problem by replacing 'utils.cent_norm(verts)' to 'Normalize()(verts)' which is in utils.py

**Comment by scho419** [comment]
> i solve this problem by replacing 'utils.cent_norm(verts)' to 'Normalize()(verts)' which is in utils.py

Did you end up being able to use this code for PointNetSeg? If you did it would be great to know how you have solved the other issues - thank you!

**Comment by Largehare** [comment]
> > i solve this problem by replacing 'utils.cent_norm(verts)' to 'Normalize()(verts)' which is in utils.py
> 
> Did you end up being able to use this code for PointNetSeg? If you did it would be great to know how you have solved the other issues - thank you!

Yes, I successfully run the code in the end. But I have forgot what other issues I had met.

**Comment by Bailey-24** [comment]
> > > i solve this problem by replacing 'utils.cent_norm(verts)' to 'Normalize()(verts)' which is in utils.py
> > 
> > 
> > Did you end up being able to use this code for PointNetSeg? If you did it would be great to know how you have solved the other issues - thank you!
> 
> Yes, I successfully run the code in the end. But I have forgot what other issues I had met.

Hi, I'm a freshman in the school. So I didn't know how to implement point cloud code.
I would be very thankful if you could send me your code about `pointnetSeg.ipynb` in the project :)

**Comment by Largehare** [comment]
Here is the link:https://www.kaggle.com/code/largehare/pointnetseg/notebook. You can have a try.

> > > > i solve this problem by replacing 'utils.cent_norm(verts)' to 'Normalize()(verts)' which is in utils.py
> > > 
> > > 
> > > Did you end up being able to use this code for PointNetSeg? If you did it would be great to know how you have solved the other issues - thank you!
> > 
> > 
> > Yes, I successfully run the code in the end. But I have forgot what other issues I had met.
> 
> Hi, I'm a freshman in the school. So I didn't know how to implement point cloud code. I would be very thankful if you could send me your code about `pointnetSeg.ipynb` in the project :) And the structure tree of the project (put the data to where and which datasets your used)

Here is the link:https://www.kaggle.com/code/largehare/pointnetseg/notebook. You can have a try.

**Comment by Largehare** [comment]
> > > > i solve this problem by replacing 'utils.cent_norm(verts)' to 'Normalize()(verts)' which is in utils.py
> > > 
> > > 
> > > Did you end up being able to use this code for PointNetSeg? If you did it would be great to know how you have solved the other issues - thank you!
> > 
> > 
> > Yes, I successfully run the code in the end. But I have forgot what other issues I had met.
> 
> Hi, I'm a freshman in the school. So I didn't know how to implement point cloud code. I would be very thankful if you could send me your code about `pointnetSeg.ipynb` in the project :) And the structure tree of the project (put the data to where and which datasets your used)

Here is the link:https://www.kaggle.com/code/largehare/pointnetseg/notebook. You can have a try.

**Comment by Bailey-24** [comment]
Thank you. I'm trying.

**Comment by Bailey-24** [comment]
Thank you.
I change the dir of the dataset, and it works.
Thanks again.
